assets/css/style.css

```diff
@@ -366,8 +367,14 @@ footer.site-footer {
     margin: 0 auto;
 }
 
-@media all and (max-width: 1500px) {
-    .list.section {
-        width: 90%;
+@media (min-width: 1200px) {
+    #list {
+        width: 80%; 
+        float: right; 
+        min-height: 50vh; 
+    }
+
+    .card-item .info-column {
+        min-width: unset !important;
```

Comment:
Do you need to use `!important` here? If you simplify your CSS selectors and use BEM notation, you will very rarely need to `!important` ;)